python/calico/felix/config.py

```diff
@@ -173,24 +169,8 @@ def __init__(self):
         self.parameters = {}
```

Comment:
Please update block comment above to reflect the new general reality, and specifically not to mention ETCD_ADDRS.